android/src/app/organicmaps/MwmWork.java

```diff
@@ -0,0 +1,52 @@
+package app.organicmaps;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.work.Worker;
+import androidx.work.WorkerParameters;
+import app.organicmaps.util.CrashlyticsUtils;
+import app.organicmaps.util.log.Logger;
+
+public abstract class MwmWork extends Worker
+{
+
+  private final Context mContext;
+  private final WorkerParameters mWorkerParameters;
+  private static final String TAG = MwmWork.class.getSimpleName();
+
+  protected abstract void onHandleWorkInitialized(@NonNull WorkerParameters workerParameters);
+
+
+  @NonNull
+  protected String getTag()
+  {
+    return getClass().getSimpleName();
+  }
+
+  public MwmWork(@NonNull Context context, @NonNull WorkerParameters workerParams)
+  {
+    super(context, workerParams);
+    this.mContext=context;
```

Comment:
Did you read our style guide? 
```suggestion
    this.mContext = context;
```

Reply:
I had read it, apologies for forgot to to do code->reformat file